### ISSUE TITLE
Missing function: grammarous#close_info_window

### ISSUE THREAD

**Issue by fszymanski** [rhysd/vim-grammarous#31]
I get this error when using `<Plug>(grammarous-close-info-window)`:
```
Error detected while processing /home/???/.config/nvim/bundle/repos/github.com/rhysd/vim-grammarous/autoload/grammarous.vim:
line    9:
E741: Value is locked: g:grammarous#root
E117: Unknown function: grammarous#close_info_window
```

**Comment by rhysd** [comment]
Did you load this plugin twice? Let me know reproducable steps.

**Comment by rhysd** [comment]
There is a little reason to lock `g:grammarous#root`. So I stopped locking it. Could you update this plugin and try the latest?

**Comment by fszymanski** [comment]
I use [dein.vim](https://github.com/Shougo/dein.vim). The plugin is loaded only once.
My mapping: `nmap <Leader>gc <Plug>(grammarous-close-info-window)`.
1) I write `hello, World!`.
2) Then execute `:GrammarousCheck`
3) The info window opens.
4) I press `<Leader>gc` (to close it) and i get this error now:
```
E117: Unknown function: grammarous#close_info_window
```
The problem is that this function (grammarous#close_info_window) does not exist.

**Comment by rhysd** [comment]
@fszymanski Thank you for the detail. I'll check it.

**Comment by rhysd** [comment]
@fszymanski I fixed it in 78f7fcb. Could you try the latest?

**Comment by fszymanski** [comment]
Works great. Thanks.
